src/Raven.Server/Documents/Handlers/TimeSeriesHandler.cs

```diff
@@ -644,6 +644,40 @@ public async Task RemoveTimeSeriesPolicy()
         }
 
         [RavenAction("/databases/*/timeseries/names/config", "POST", AuthorizationStatus.ValidUser)]
```

Comment:
It looks like your method landed in between other attribute and ConfigTimeSeriesNames